### PR TITLE
Add AA Examples

### PR DESCRIPTION
**Pull request by wsdt**
## Overview

Add 2 AA examples (high level and low level api)

Note: Used the local packages (which runs through successfully) - but replaced with version numbers for remote packages to be a viable example. Because of sdk change etc. the examples only work with the upcoming release once it is merged. Due to `^` version annotation it would automatically use the newest version then. 

## Changes

- Add high level example
- Add low level examples
- Add block range config to bundler sdk (as needed to avoid blockrange exceeded error)

## Testing

Script & test running through successfully on Boba BNB testnet.

### CODE REVIEW

**Review by InoMurko — APPROVED** [review]
Goodio!

File: boba_examples/aa-high-level-example/scripts/index.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
+"use strict";
```
Comment (souradeep-das) [inline boba_examples/aa-high-level-example/scripts/index.js]: tsc generated file?

File: boba_examples/aa-high-level-example/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Account Abstraction example
+
+Example for Boba BNB testnet to send userOperation to a wallet through the bundler (high level api).
+This example conducts a simple ETH transfer.
```
Comment (souradeep-das) [inline boba_examples/aa-high-level-example/README.md]: simple contract call?

File: boba_examples/aa-low-level-example/scripts/index.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
+"use strict";
```
Comment (souradeep-das) [inline boba_examples/aa-low-level-example/scripts/index.js]: tsc generated file?

File: boba_examples/aa-low-level-example/scripts/index.ts [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    to: account,
+  })
+
+  const accountAPI = new SimpleAccountAPI({
+    provider: local_provider,
+    entryPointAddress,
+    owner: l2Wallet,
+    accountAddress: account,
+  })
+
```
Comment (souradeep-das) [inline boba_examples/aa-low-level-example/scripts/index.ts:65]: some comment logs on the intermediary steps could be useful to the user, when they are waiting for the script to run? 
for ex - generating userOp, signing etc

**Review by souradeep-das — APPROVED** [review]
🚀

File: boba_examples/aa-high-level-example/scripts/index.js (hunk 1 above)
Comment (wsdt) [reply boba_examples/aa-high-level-example/scripts/index.js]: yes!

**Pull request merged**